ocaml/testsuite/tests/typing-modes/val_modalities.ml

```diff
@@ -277,7 +277,7 @@ module Close_over_value :
 
 module Close_over_value_monadic = struct
   module M = struct
-    let r @ uncontended = "hello"
+    let r @ uncontended = fun x -> x
```

Comment:
Might be better to use `int ref`  (because arrows will cross contention)

Reply:
Unfortunately the currently mode-checking wrt mutable fields is too strict. Can you add a `(* CR mode-crossing: this should use mutable record *)` and I will fix in later PRs?

Reply:
Done